pipelines.py

```diff
@@ -176,16 +176,38 @@ def _preprocessing_multitask_generator(config, is_train, use_patching):
     return loader
 
 
-def mask_postprocessing(model, config, save_output=False):
-    mask_resize = Step(name='mask_resize',
-                       transformer=ResizerStream() if config.execution.stream_mode else Resizer(),
-                       input_data=['input'],
-                       input_steps=[model],
-                       adapter={'images': ([(model.name, 'multichannel_map_prediction')]),
-                                'target_sizes': ([('input', 'target_sizes')]),
-                                },
-                       cache_dirpath=config.env.cache_dirpath,
-                       save_output=save_output)
+def mask_postprocessing(loader, model, config, save_output=False):
+    if config.postprocessor.crf.apply_crf:
+        dense_crf = Step(name='dense_crf',
+                         transformer=DenseCRFStream(**config.postprocessor.crf) if config.execution.stream_mode \
+                             else DenseCRF(**config.postprocessor.crf),
+                         input_steps=[loader, model],
+                         adapter={'images': ([(model.name, 'multichannel_map_prediction')]),
+                                  'org_images_gen': ([(loader.name, 'datagen')]),
```

Comment:
@taraspiotr org_ -> original_ or raw or something more descriptive makes it easier to read